ob1k-cache/src/main/java/com/outbrain/ob1k/cache/memcache/folsom/MemcachedClient.java

```diff
@@ -79,12 +79,29 @@ public MemcachedClient(final MemcacheClient<V> folsomClient, final CacheKeyTrans
 
   @Override
   public ComposableFuture<Boolean> setAsync(final K key, final V value) {
+    return setAsync(key, value, expirationSeconds);
+  }
+
+  public ComposableFuture<Boolean> setAsync(final K key, final V value, final int expirationSeconds) {
```

Comment:
As we discussed, we need variable expiration & setIfAbsent to migrate away from spy.
You want this in a different PR?